script/update_rubygems_and_install_bundler

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# This file was generated on 2019-01-03T20:34:23+00:00 from the rspec-dev repo.
```

Comment:
I don't see it on rspec-dev at the moment https://github.com/rspec/rspec-dev/tree/master/travis/script do we want to also do this change on other RSpec repos?

Also I see those functions but not where it is called? 🤔

Reply:
It's a WIP until I see it's working :p (also this is the worst repo to pick to review as its the most out of sync!)